wallet-new/server/Cardano/Wallet/API/V1/LegacyHandlers/Wallets.hs

```diff
@@ -23,15 +23,14 @@ handlers :: ( HasConfigurations
             , HasCompileInfo
             )
          => ServerT Wallets.API MonadV1
-handlers =   newWallet
-        :<|> listWallets
-        :<|> (\walletId ->
-                     updatePassword walletId
-                :<|> deleteWallet walletId
-                :<|> getWallet walletId
-                :<|> updateWallet walletId
-                :<|> Accounts.handlers walletId
-             )
+handlers = newWallet
+    :<|> listWallets
+    :<|> ( updatePassword
+        :<|> deleteWallet
+        :<|> getWallet
+        :<|> updateWallet
+        :<|> Accounts.handlers
+    )
```

Comment:
I've moved the Account handlers out, to be virtually separated from the wallet handlers in order to avoid nesting tags. 
Expect a merge-conflict here with CBR-102. Nothing hard to resolve though.